README.md

```diff
@@ -165,4 +165,7 @@ If this repository isn't quite what you're looking for, try:
 
 ## Development
 
-To create a release on github, simply push a tag beginning with "v".
\ No newline at end of file
+To create a release on github, merge a commit which removes "-snapshot" from [VERSION](VERSION) (and, optionally,
+bumps the major version if there has been an incompatible change), then push a tag beginning with "v".
```

Comment:
the AZP pipeline is only building pushes to master. The real trigger is not the tag, but the tagged commit being pushed to master.

Reply:
I'm going to confirm this experimentally by merging this PR and then pushing a tag.

Reply:
I waited for the [master CI job](https://dev.azure.com/projectriff/pivotal-image-relocation/_build/results?buildId=3236) to complete after merging this PR. Then I issued the following commands:
```
git pull origin master
git log # to confirm 03c983ad4d2321ec560136c756865dd2bb84138f is at HEAD
git tag v0.7
git push origin --tags
```
I then observed the following in the [actions tab](https://github.com/pivotal/image-relocation/actions):
![Screenshot 2019-11-05 at 08 50 02](https://user-images.githubusercontent.com/94953/68192881-f49d0d80-ffa9-11e9-8689-8748aa86140e.png)
When this completed, the [v0.7 release](https://github.com/pivotal/image-relocation/releases/tag/v0.7) had been correctly populated with binaries etc.:
![Screenshot 2019-11-05 at 08 55 47](https://user-images.githubusercontent.com/94953/68193013-38901280-ffaa-11e9-8b55-6095fe0905d9.png)